src/main/java/com/iexec/blockchain/config/PublicConfigurationController.java

```diff
@@ -42,7 +42,14 @@ public PublicConfigurationController(ChainConfig chainConfig) {
         final Integer blockTime = chainConfig.getBlockTime();
         final PublicChainConfig publicChainConfig = PublicChainConfig
                 .builder()
-                .blockTime(Duration.ofSeconds(blockTime))
+                .chainId           (chainConfig.getChainId())
```

Comment:
Let's stay consistent with the rest of the code
```suggestion
                .chainId(chainConfig.getChainId())
```